modules/ui/src/app/pages/risk-assessment/risk-assessment.store.ts

```diff
@@ -110,9 +110,7 @@ export class RiskAssessmentStore extends ComponentStore<AppComponentState> {
     return trigger$.pipe(
       delay(10),
       tap(() => {
-        this.focusManagerService.focusFirstElementInContainer(
-          window.document.querySelector('.content-empty')
-        );
+        this.focusManagerService.focusFirstElementInContainer();
```

Comment:
It can set the focus to another element out of this component, for example to the callout link. So, please check such scenario.